roomserver/alias/alias.go

```diff
@@ -96,12 +96,14 @@ func (r *RoomserverAliasAPI) GetRoomIDForAlias(
 		return err
 	}
 
-	// No rooms found locally, try our application services by making a call to
-	// the appservice component
-	aliasReq := appserviceAPI.RoomAliasExistsRequest{Alias: request.Alias}
-	var aliasResp appserviceAPI.RoomAliasExistsResponse
-	if err = r.AppserviceAPI.RoomAliasExists(ctx, &aliasReq, &aliasResp); err != nil {
-		return err
+	if len(roomID) == 0 {
```

Comment:
I might make this `if roomID == ""` just to be clear that roomID is a string versus a Slice that could also have zero length.